plugins/locate.py

```diff
@@ -16,15 +16,20 @@ def check_status(status):
         Returns None if no errors found """
     if status == 'REQUEST_DENIED':
         return 'The geocode API is off in the Google Developers Console.'
-    elif status == 'ZERO_RESULTS':
```

Comment:
Could this be done with a dict instead?

Reply:
It could, but I'm not changing logic in this PR.